pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -23,25 +23,26 @@ const wxChar* Panels::SpeedHacksPanel::GetEEcycleSliderMsg( int val )
 {
 	switch( val )
 	{
+		switch( val )
+	{
```

Comment:
Nope, that was done on Git online editor. should work fine now :) 

I'm not really sure what to do with it though , the users at poll thread seems to have lots of weird ideas @_@